jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -259,7 +259,11 @@ YUI.add('user-profile-model-list', function() {
           return null;
         }
       }
-      const lastConnection = model.lastConnection || '--';
+      const region = model.region || 'no region';
+      let owner = '--';
```

Comment:
Agreed, we should always have an owner. The defensiveness is due to pre-existing code. I'll add a comment.